uitools/cpp/Esri/ArcGISRuntime/Toolkit/Internal/GeoViews.h

```diff
@@ -52,4 +46,30 @@ namespace Toolkit
 
 #endif
 
+namespace Esri {
+namespace ArcGISRuntime {
+namespace Toolkit {
+  inline Map* getGeoModel(MapViewToolkit* mapView)
+  {
+    return mapView ? mapView->map() : nullptr;
+  }
+
+  inline Scene* getGeoModel(SceneViewToolkit* sceneView)
```

Comment:
any reason we don't implement these in a cpp?

Reply:
Currently this is  a header-only file at the moment. These two methods could be added to a C++, but no other file can be moved ATM.

I'd like to keep them inlined (for now), as these are a cut-and-dry-case of when inlining is appropriate, IMO. But the long-term goal is to get rid of these entirely and rely on a `getGeoModel` coming from the API.